_data/en/store.yml

```diff
@@ -125,6 +125,9 @@ items:
     helsinkisystems-budgetwatcher:
         title: Budget Watcher (watch project budgets and send notifications)
         intro: Get notified whenever a certain amount of your projects budget is used
+    helsinkisystems-lexoffice:
+        title: Lexoffice (automatically create invoices in Lexoffice from invoices in Kimai)
```

Comment:
The title is something like "Lexoffice Invoice" or Lexoffice Invoice Sync" right?
The intro contains the description. 
Please remove the (rest) from the title and while you are on it: remove it from Budget watcher as well please in german and english.